### PR TITLE
CORTX-33669 critical/minor Codacy fixes related to style and security

### PR DESCRIPTION
**Pull request by seagate-sarang-sawant**
Problem: Reduce the number of Critical and minor Codacy issues. Identify Test coverage of modules where security and other fixes are done.  
Solution:  a) Code/doc comment changes in the files which had maximum issues and which does not require logical change or refactoring. 
                b) Tested utils security fix.
                c) False positive and conflicting Codacy rules will be disabled.

### CODE REVIEW

**Comment by vaibhavparatwar** [comment]
retest this please

**Comment by vaibhavparatwar** [comment]
@seagate-sarang-sawant please add proper commit message and signoff so DCO check is passed.

**Comment by vaibhavparatwar** [comment]
@seagate-sarang-sawant  Codacy is failing.. with "solved 23 and added 3" message. also

**Comment by vaibhavparatwar** [comment]
I see Hare Sanity and 1N/3N deployment is working fine for this PR.. lets debug premerge related issue.

**Review by vaibhavparatwar — APPROVED** [review]

**Review by Shreya-18 — APPROVED** [review]

**Comment by vaibhavparatwar** [comment]
@mssawant need your review on this. cc @seagate-sarang-sawant

**Comment by mssawant** [comment]
@vaibhavparatwar, @seagate-sarang-sawant, is hare-sanity failure known?

**Comment by mssawant** [comment]
retest this please

File: hax/hax/motr/planner.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def is_current(cmd: BaseMessage, st: State) -> bool:
     def _get_increased_group(self, current: int) -> int:
         ''' Returns the next valid group_id number by the given current value.
```
Comment (d-nayak) [inline hax/hax/motr/planner.py:252]: """ ?

File: hax/hax/motr/planner.py [hunk 2]
```diff
@@ -251,7 +251,7 @@ def is_current(cmd: BaseMessage, st: State) -> bool:
...
 
-            Performs no side effects.
+        Performs no side effects.
         '''
```
Comment (d-nayak) [inline hax/hax/motr/planner.py:255]: """ ?

File: hax/hax/motr/planner.py [hunk 3]
```diff
@@ -353,8 +353,9 @@ def has(cmd_type: Type[BaseMessage]) -> bool:
         return False
 
     def _assign_group(self, cmd: BaseMessage) -> Tuple[BaseMessage, bool]:
-        ''' Sets the correct group_id to the command. Side effect: updates
-        self.state.
+        '''Sets the correct group_id to the command.
```
Comment (d-nayak) [inline hax/hax/motr/planner.py:356]: """ ?

File: hax/test/integration/testutils.py [hunk 4]
```diff
@@ -35,11 +35,11 @@ def trace_call(fn):
     '''
```
Comment (d-nayak) [inline hax/test/integration/testutils.py:35]: """ ?

File: hax/test/integration/testutils.py [hunk 5]
```diff
@@ -35,11 +35,11 @@ def trace_call(fn):
...
     Decorator that 'logs' the historical sequence of all the methods of the
     given class.
-
     The implementation assumes that
     1. the decorator will decorate methods of a class (not static functions).
     2. the class has field `traces: List[Invocation]`
     '''
```
Comment (d-nayak) [inline hax/test/integration/testutils.py:41]: """ ?

Comment (seagate-sarang-sawant) [reply hax/test/integration/testutils.py:41]: There are single quote docstring comment  like '''  in most of the modules. We need to convert them into double quote doc string comment and will be taken in next part. The rule pertaining to avoiding usage of ''' must have been be disabled.

File: hax/hax/motr/planner.py (hunk 2 above)
Comment (d-nayak) [reply hax/hax/motr/planner.py:255]: Will be taken up separately, as per @seagate-sarang-sawant

File: hax/hax/motr/planner.py (hunk 3 above)
Comment (d-nayak) [reply hax/hax/motr/planner.py:356]: Will be taken up separately, as per @seagate-sarang-sawant

File: hax/test/integration/testutils.py (hunk 4 above)
Comment (d-nayak) [reply hax/test/integration/testutils.py:35]: Will be taken up separately, as per @seagate-sarang-sawant

File: hax/test/integration/testutils.py (hunk 5 above)
Comment (d-nayak) [reply hax/test/integration/testutils.py:41]: Will be taken up separately, as per @seagate-sarang-sawant

**Review by d-nayak — APPROVED** [review]

**Comment by mssawant** [comment]
retest this please

**Review by pavankrishnat — APPROVED** [review]
@seagate-sarang-sawant, please attach test output

**Pull request merged**